sys_conn_oob_test.go

```diff
@@ -310,6 +311,16 @@ var _ = Describe("OOB Conn Test", func() {
 		})
 	})
 
+	Context("serializing packet info", func() {
+		It("treats IPv4-mapped IPv6 addresses as IPv4", func() {
```

Comment:
Does this test fail on FreeBSD without your fix?

Reply:
In fact it should on any platform, because the packet will look different. Some are however insensitive to that and will happily send it.

Reply:
Interestingly, on Windows it _doesn't_ fail. On macOS, FreeBSD and Ubuntu it does. This is very curious, so I will try to investigate further.

Reply:
Ah, these tests are skipped on Windows...